doc/grass_database.html

```diff
@@ -122,7 +127,7 @@ <h3>GRASS Mapsets</h3>
 
 <p>
 Mapsets can be copied and moved as directories, however only when it is clear
-that the projections of both Locations
+that the projections of both projects
 (as reported by <a href="g.proj.html"><em>g.proj</em></a>)
 match each other. Since this is sometimes hard to to establish,
```

Comment:
```suggestion
that their projections 
(as reported by <a href="g.proj.html"><em>g.proj</em></a>)
match. Since this is sometimes hard to to establish,
```